make.html.markdown

```diff
@@ -222,7 +222,7 @@ report:
 ifeq ($(sport),tennis)
 	@echo 'game, set, match'
 else
-	@echo 'They think it's all over; it is now'
+	@echo "They think it's all over; it is now"

```

Comment:
The ' in  "it's" closes the quotation and then the ";" finishes the statement and then " it is now" throws an error.